src/extras/ImageUtils.js

```diff
@@ -47,6 +47,8 @@ const ImageUtils = {
 
 		if ( canvas.width > 2048 || canvas.height > 2048 ) {
 
+			console.warn( 'THREE.ImageUtils.getDataURL: Image converted to jpg for performance reasons', image );
```

Comment:
Is it necessary to log the image object here? I remember there were performance issues in the past when doing this.

Reply:
Hmm... Maybe `image.src` instead?

Reply:
`image.src` won't work if image is of type `ImageBitmap`

Reply:
I think we can start like so and remove the property if somebody complains. Unfortunately, I can't find the related discussion anymore so it's probably safe anyway.

Reply:
Sounds good!